index.html

```diff
@@ -35,8 +36,8 @@ <h2>Company Name?</h2>
                     </a>
                     <button type="monbutton" class="btn btn-secondary dayLink" data-id="Monday"></button>
                     <ul class="list-group">
-                        <li class="list-group-item">Employees assigned: 99</li>
-                        <li class="list-group-item">Tasks Assigned: 99</li>
+                        <li class="list-group-item">Employees assigned: 99</option>
```

Comment:
all of these <li> tags have a </option> closing tag - I've looked online, does this actually work? seems like you still need to match closing tag